docs/rules/explicit-length-check.md

```diff
@@ -35,13 +35,13 @@ if (string.length < 1) {}
 ### Pass
 
 ```js
-if (array.length !== 0) {}
+if (array.length === 0) {}
```

Comment:
I believe these two were typos because this section talks about **zero** comparisons. So I fixed them.